docs/guide/docs/channels/smooch.md

```diff
@@ -13,32 +13,45 @@ title: Smooch (Sunshine Conversations)
 
 - Create a [smooch.io](https://smooch.io/) account (you will need a business email)
 
-## Steps
+## Setup
 
-### Create the app on Smooch
+### Smooch App
+Firstly, you need to create an app on smooch. To do so, please follow the steps below:
 
 1. On the `app.smooch.io` home page, click on `Create new app`
 
 2. Enter a name for your app and click `Create App`
 
-3. Connect a channel to your app (ex: telegram)
+3. Connect a channel to your app (Telegram, Whatsapp, or any other listed channel)
 
-4. You should now see a channel in `Overview` section of your app
+4. You should see a channel in the `Overview` section of your app
 
-### Connect Smooch to botpress
+### Botpress Connection
 
-1. Go in the `Settings` section of your app
+1. Go to the `Settings` section of your app
 
-2. At the bottom you will find an `API Keys` menu. Create a new API key
+2. At the bottom, you will find an `API Keys` menu. Create a new API key
 
 3. You will need the `id` (the one that starts with app\_) and `secret` key to setup your bot
 
-### Configure your bot
+## Configuration
 
 1. Edit `data/bots/YOUR_BOT_ID/config/channel-smooch.json` (or create it) and set
 
 - enabled: Set to `true`
-- keyId: Paste the `id` of your key from `Settings` section
-- secret: Paste the `secret` of your key from `Settings` section
-
-2. Restart Botpress
+- keyId: Paste the `id` of your key from the `Settings` section
+- secret: Paste the `secret` of your key from the `Settings` section
+
+The resulting json will appear as follows:
+
+```json
+{
+  "$schema": "../../assets/modules/channel-smooch/config.schema.json",
+  "enabled": true,
+  "keyId": "app_5f079386f6480a000c121f71",
```

Comment:
I hope those are only placeholders

Reply:
Should we use something like [YOUR_KEY_ID] instead of an actual key? I remember seeing something like this in other documentation. Find this helpful.